PluginCore/PluginCore/Helpers/PathHelper.cs

```diff
@@ -19,7 +20,7 @@ public static String BaseDir
         {
             get
             {
-                if (PluginBase.MainForm.StandaloneMode) return AppDir;
+                if (PluginBase.MainForm.StandaloneMode) return Path.GetDirectoryName(GetAssemblyPath(Assembly.GetExecutingAssembly()));
```

Comment:
That's a bit subjective... It's normal that the directory of the application is where the executable file resides, while the base directory is not necessarily the same as the one where the application resides.

The problem here is that we don't have full control or who is running the tests and where from, so the application making use of the FD libraries is not in the same folder as the base directory where all of the needed files are.